recbole/data/dataloader/general_dataloader.py

```diff
@@ -3,9 +3,9 @@
 # @Email  : houyupeng@ruc.edu.cn
 
 # UPDATE
-# @Time   : 2020/9/9, 2020/9/29, 2021/7/15, 2022/7/6
-# @Author : Yupeng Hou, Yushuo Chen, Xingyu Pan, Gaowei Zhang
-# @email  : houyupeng@ruc.edu.cn, chenyushuo@ruc.edu.cn, xy_pan@foxmail.com, zgw15630559577@163.com
+# @Time   : 2022/7/8, 2020/9/9, 2020/9/29, 2021/7/15, 2022/7/6
```

Comment:
Time, author and email information should be arranged in chronological order and placed at the end.